tox.ini

```diff
@@ -92,4 +92,13 @@ description = Run integration tests
 deps =
     pytest
 commands =
-    pytest -v --tb native --ignore={[vars]tst_dir}unit --log-cli-level=INFO -s {posargs}
+    pytest -v --tb native --ignore={[vars]tst_dir}unit \
+        --ignore={[vars]tst_dir}integration/test_dnf.py \
+        --log-cli-level=INFO -s {posargs}
+
+[testenv:integration-but-with-cleantest]
```

Comment:
I will change to a shorter name. I just added a separate tox env because I knew using cleantest for testing would be a topic for debate. I did not want to muck up the current integration-tests tox env while we hammer out the details behind how we should go about testing the dnf charm library.

Reply:
Named changed to `integration-cleantest` in https://github.com/canonical/operator-libs-linux/pull/75/commits/5fea11961abc6efd241d8b171de3ca181f517513